doc/source/apis.rst

```diff
@@ -5,19 +5,72 @@ Working on Android
 This page gives details on accessing Android APIs and managing other
 interactions on Android.
 
+Storage paths
+-------------
+
+If you want to store and retrieve data, you shouldn't just save to
+the current directory, and not hardcode `/sdcard/` or some other
+path either - it might differ per device.
+
+Instead, the `android` module which you can add to your `--requirements`
+allows you to query the most commonly required paths::
+
+      from android.storage import app_storage_path
+      settings_path = app_storage_path()
+
+      from android.storage import primary_external_storage_path
+      primary_ext_storage = primary_external_storage_path()
+
+      from android.storage import most_portable_storage_path
+      portable_storage = most_portable_storage_path()
+
+`app_storage_path()` gives you Android's so-called "internal storage".
+It compares best to the AppData directory on Windows.
```

Comment:
We may say that unlike the two external storage, this one gets wiped out when the application is removed.